__tests__/components/forecast-summaries.test.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import ForecastSummary from '../../components/forcast-summary';
+import ForecastSummaries from '../../components/forecast-summaries';
+import Enzyme from 'enzyme';
+
+const forecasts = [
+  {
+    date: 'date1',
+    description: 'desc1',
+    icon: 'icon1',
+    temperature: {
+      max: 999
+    }
+  },
+  {
+    date: 'date2',
+    description: 'desc2',
+    icon: 'icon2',
+    temperature: {
+      max: 909
+    }
+  }
+];
+it('renders the correct amount of summary components from the parent', () => {
```

Comment:
If you're creating the same wrapper more then once, you may consider using the `beforeEach()`.